terraform/email.tf

```diff
@@ -0,0 +1,94 @@
+locals {
+  communication_service_name                 = "ACS-CDT-PUB-VIP-CALITP-${local.env_letter}-001"
+  data_location                              = "United States"
+  custom_domain_name                         = "benefits.calitp.org"
+  azure_communication_connection_string_name = "azure-communication-connection-string"
+  azure_communication_from_email_name        = "azure-communication-from-email"
+  # Determine the correct sender domain based on the environment.
+  # If is_prod is true, use the custom domain; otherwise, use the Azure-managed domain.
+  sender_domain = local.is_prod ? azurerm_email_communication_service_domain.custom[0].mail_from_sender_domain : azurerm_email_communication_service_domain.azure_managed[0].mail_from_sender_domain
```

Comment:
this line got me thinking...

do y'all get an indication in VS Code for formatting errors in .tf files? i do not. outside the container i confirmed that the [`hashicorp.terraform`](https://github.com/cal-itp/benefits/blob/97132f78ef47f3ef68611f405c058f9c08674f5e/.devcontainer/devcontainer.json#L29) extension needs `terraform` to be installed too.

seems like the juice would be worth the squeeze to do that inside the container and to enable auto-formatting, but i won't rat-hole on it unless y'all agree its worth creating a ticket for slush time.

Reply:
Yeah I think we've discussed installing `terraform` in the devcontainer before, I can't remember where those discussions are. I also can't remember why we haven't done it yet... Slush ticket sounds good :+1: